packages/sdk/src/evm/core/classes/appuri.ts

```diff
@@ -0,0 +1,51 @@
+import { FEATURE_APPURI } from "../../constants/thirdweb-features";
+import { DetectableFeature } from "../interfaces/DetectableFeature";
+import { TransactionResult } from "../types";
+import { ContractWrapper } from "./contract-wrapper";
+import type { AppURI } from "@thirdweb-dev/contracts-js";
+
+/**
+ * Have an official Application URI for this contract.
+ * @remarks Configure an official Application URI for this contract.
+ * @example
+ * ```javascript
+ * const contract = await sdk.getContract("{{contract_address}}");
+ * const appURI = await contract.appURI.get();
```

Comment:
The namespace in this code example doesn't match the actual namespace that you've setup on the contract in the `smart-contract.ts` file (namespace is case sensitive)

Reply:
fixed in latest

Reply:
Also remember to fix the namespace here if you change it above